src/adminserver/systeminfo/imagestorage/filesystem/driver.go

```diff
@@ -43,6 +44,11 @@ func (d *driver) Name() string {
 // Cap returns the capacity of the filesystem storage
 func (d *driver) Cap() (*storage.Capacity, error) {
 	var stat syscall.Statfs_t
+	if _, err := os.Stat(d.path); os.IsNotExist(err) {
+		// Return zero value if the path does not exist.
```

Comment:
better to print a warning in case the user specifies a wrong image store dir

Reply:
yep sure! Thanks.